pkg/presets/presets.go

```diff
@@ -10,6 +10,8 @@ var (
 	AnnounceURL, _        = url.Parse("https://cid.contact/announce")
 	IndexingServiceDID, _ = did.Parse("did:web:indexer.storacha.network")
 	IndexingServiceURL, _ = url.Parse("https://indexer.storacha.network")
+	UploadServiceURL, _   = url.Parse("https://up.storacha.network")
+	UploadServiceDID, _   = did.Parse("did:web:up.storacha.network")
```

Comment:
To Reviewer: please validate I have defined these correctly, based on code I've seen in other repos.

Reply:
yes, to my knownledge